style.css

```diff
@@ -16,6 +16,95 @@ body {
   margin: 0;
 }
 
+/* popup style */
+
+.close {
+  align-self: flex-end;
+  padding: 5px;
+}
+
+.popup {
```

Comment:
- [ ] The popup needs some modification so it is as close as possible to the Figma design

- Your design:
![Screenshot_20220513_071451](https://user-images.githubusercontent.com/25227022/168222849-8133a91e-0802-4699-9ee8-40c324d693fd.png)

- Figma design:
![Screenshot_20220513_071507](https://user-images.githubusercontent.com/25227022/168222873-6286a02c-5126-42ca-8ff4-212f84be7de8.png)